mobile/src/common/enums/ui/ui.ts

```diff
@@ -1,2 +1,3 @@
+export { AppButtonVariant } from './app-button-variant';
```

Comment:
Rename this file to app-button-variant.enum ?

Reply:
use type instead of enum for this

Reply:
any advice how i can refactor theme or button styles with type instead of enum ?

Reply:
Forgot about this, yes, you was right, then it should be enum, and rename it to ButtonVariant and filename should be button-variant.enum.ts

Reply:
App prefix for more common values